includes/js/item.js

```diff
@@ -4,20 +4,25 @@ http://projekt406.ee/codeblog
 17.12.2012
 */
 
-function item() {
-	this.X = 0;
-	this.Y = 0;
-	this.color = '#3a9dc2';
-	this.type = 0; // item type for use inside the system
-	this.name = ''; // name for reference
-	this.sprite = null;
+const ITEM_HOE = 11;
 
-	this.rotation = 0;
-	this.animation = false;
-	this.animationSpeed = 2;
-	this.draw = function(board, canvas) {
+class Item {
```

Comment:
Why just make item into a class? It would make sense to have everything as a class then (player and board as well)

Reply:
Oupsie, I meant to only include the first commit #71c355f in the PR and the following commits, where mostly for me to fiddle around with the code to figure things out. But you are right, just having item and tile as a class and player and board a function is inconsistent.